### PR TITLE
tdrop: init at 2018-11-13

### PR DESCRIPTION
**Pull request by wedens**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/tdrop/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{ stdenv, lib, fetchFromGitHub, makeWrapper
+, xwininfo, xdotool, xprop }:
+
+let
+  version = "2019-01-02";
+in
+stdenv.mkDerivation {
```
Comment (worldofpeace) [inline pkgs/applications/misc/tdrop/default.nix]:
```suggestion
stdenv.mkDerivation rec {
```

This allows you to self reference so you can move `version` into the package description.

File: pkgs/applications/misc/tdrop/default.nix [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{ stdenv, lib, fetchFromGitHub, makeWrapper
+, xwininfo, xdotool, xprop }:
+
+let
+  version = "2019-01-02";
```
Comment (worldofpeace) [inline pkgs/applications/misc/tdrop/default.nix]: Should be formatted
```suggestion
  version = "unstable-2018-11-13";
```

File: pkgs/applications/misc/tdrop/default.nix [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+in
+stdenv.mkDerivation {
+  name = "tdrop-${version}";
```
Comment (worldofpeace) [inline pkgs/applications/misc/tdrop/default.nix]: Possibly use `pname`.

**Comment by worldofpeace** [comment]
@wedens Also change the version in your commit msg :smile:

**Comment by wedens** [comment]
Done 🙂

**Review by worldofpeace — APPROVED** [review]
Looks good, executed and I can see the help menu. Though I haven't tested if it functions properly.

**Comment by dywedir** [comment]
Thanks!

**Pull request merged**